openedx/core/lib/xblock_utils.py

```diff
@@ -240,7 +240,7 @@ def add_staff_markup(user, has_instructor_access, disable_staff_debug_info, bloc
         else:
             return frag
 
-    if isinstance(block, SequenceModule):
+    if isinstance(block, SequenceModule) or getattr(block, 'HIDDEN', False):
```

Comment:
@cpennington One issue I ran into was how to disable the wrappers which add extra html to the empty div of the `HiddenModule`. For now I have added an attribute on the `HiddenModule`. Is there a better pattern for this?